src/CommunityToolkit.Maui/Controls/Avatar.cs

```diff
@@ -0,0 +1,557 @@
+using System.ComponentModel;
```

Comment:
Please organize elements in this file according to the following StyleCop rules:

```
According to the [StyleCop  Rules Documentation][1] the ordering is as follows.

Within a class, struct or interface: (SA1201 and SA1203)

 - Constant Fields
 - Fields
 - Constructors
 - Finalizers (Destructors)
 - Delegates
 - Events
 - Enums
 - Interfaces (*interface implementations*)
 - Properties
 - Indexers
 - Methods
 - Structs
 - Classes

Within each of these groups order by access: (SA1202)

 - public
 - internal
 - protected internal
 - protected
 - private

Within each of the access groups, order by static, then non-static: (SA1204)

 - static
 - non-static

Within each of the static/non-static groups of fields,  order by readonly, then non-readonly : (SA1214 and SA1215)

 - readonly
 - non-readonly

An unrolled list is 130 lines long, so I won't unroll it here. The methods part unrolled is:

 - public static methods
 - public methods
 - internal static methods
 - internal methods
 - protected internal static methods
 - protected internal methods
 - protected static methods
 - protected methods
 - private static methods
 - private methods
```

https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1201.md

Reply:
I don't currently use StyleCop, but will install the extension and endeavour to meet the standards.